services/server/deps.go

```diff
@@ -76,3 +78,9 @@ type supervisordService interface {
 type telemetryService interface {
 	DistributionMethod() serverpb.DistributionMethod
 }
+
+// checksService is a subset of methods of checks.Service used by this package.
+// We use it instead of real type for testing and to avoid dependency cycle.
+type checksService interface {
```

Comment:
we have a lot of checkers and it may confuse

Reply:
Renamed to `securityChecksService`